v0/destinations/pardot/util.js

```diff
@@ -100,69 +98,9 @@ const responseHandler = destinationResponse => {
   };
 };
 
-/**
- * depricating: handles proxying requests to destinations from server, expects requsts in "defaultRequestConfig"
- * note: needed for test api
- * @param {*} request
- * @returns
- */
-const pardotProxyRequest = async request => {
-  const { body, method, params, endpoint } = request;
-  let { headers } = request;
-  let data;
-  let payload;
-  let payloadFormat;
-  Object.entries(body).forEach(([key, value]) => {
-    if (!isEmpty(value)) {
-      payload = value;
-      payloadFormat = key;
-    }
-  });
-
-  switch (payloadFormat) {
-    case "JSON_ARRAY":
-      data = payload.batch;
-      // TODO: add headers
-      break;
-    case "JSON":
-      data = payload;
-      headers = { ...headers, "Content-Type": "application/json" };
-      break;
-    case "XML":
-      data = `${payload}`;
-      headers = { ...headers, "Content-Type": "application/xml" };
-      break;
-    case "FORM":
-      data = new URLSearchParams();
-      data.append("format", "json");
```

Comment:
@saikumarrs 
This addition of `format` key is needed for `Pardot`. Except this, everything else is the same. Kindly please add this back at least till we decide on how to handle such case(s) going forward

Reply:
Thanks. Addressed.